integration

```diff
@@ -307,6 +307,7 @@
   "delphiki/hass-pronote",
   "denpamusic/homeassistant-plum-ecomax",
   "denysdovhan/ha-check-weather",
+  "dermotduffy/hass-web-proxy-integration",
```

Comment:
Did you abandon this already?

Reply:
Only outstanding PRs are chore dependency update PRs. Both are reasonably feature complete for the initial usecase, in this case this is actively used by [Advanced Camera Card](https://github.com/dermotduffy/advanced-camera-card) which is certainly under active usage/development.

Reply:
I've now cleared the dependabot PRs.